clib/test/main.cc

```diff
@@ -18,7 +18,7 @@
  */
 #include <gtest/gtest.h>
 
-int main(int argc, char* argv[], char** env) {
+int main(int argc, char* argv[], char** env [[maybe_unused]]) {
```

Comment:
Isn't it possible to simply remove env argument?

Reply:
I will remove